packages/client.api/etc/client.api.report.api.md

```diff
@@ -0,0 +1,397 @@
+## API Report File for "@osdk/client.api"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
```

Comment:
Generated file which lets us see:
- what was exported
- what was referenced without being exported
- what external packages are part of our public api
- which types are undocumented
- which types are public/internal/beta